templates/Deepcell_Postprocessing.ipynb

```diff
@@ -14,12 +14,16 @@
    "outputs": [],
```

Comment:
<p>Remember to remove the <code>sys</code> imports and <code>sys.path.append(...)</code> statements on pushed notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>